### PR TITLE
Added usage of benchmarksAnalyzer for analysis of performance, so rem…

### PR DESCRIPTION
**Pull request by LepilkinaElena**
…oved extra code from benchmark application and gradle.

### CODE REVIEW

File: performance/buildSrc/src/main/kotlin/MPPTools.kt [hunk 1]
```diff
@@ -77,6 +78,15 @@ fun createJsonReport(projectProperties: Map<String, Any>): String {
     return report.toJson()
 }
 
+// Find file with set name in directory.
+fun findFile(fileName: String, directory: String): String? {
+    File(directory).walkBottomUp().forEach {
```
Comment (olonho) [inline performance/buildSrc/src/main/kotlin/MPPTools.kt]: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/find.html

File: performance/gradle.properties [hunk 2]
```diff
@@ -8,4 +8,5 @@ nativeTextReport = nativeReport.txt
 jvmTextReport = jvmReport.txt
 nativeJson = nativeReport.json
 jvmJson = jvmReport.json
-
+analyzerTool = benchmarksAnalyzer.kexe
```
Comment (olonho) [inline performance/gradle.properties]: For Windows extension is .exe

File: performance/buildSrc/src/main/kotlin/MPPTools.kt [hunk 3]
```diff
@@ -61,6 +63,13 @@ fun defaultHostPreset(
         throw Exception("Host OS '$hostOs' is not supported in Kotlin/Native ${subproject.displayName}.")
 }
 
+fun getNativeProgramExtension(subproject: Project): String = when {
+    isMacos -> ".kexe"
+    isLinux -> ".kexe"
+    isWindows -> ".exe"
+    else -> ""
```
Comment (olonho) [inline performance/buildSrc/src/main/kotlin/MPPTools.kt]: `else -> TODO("Unknown host")`

File: performance/buildSrc/src/main/kotlin/MPPTools.kt [hunk 4]
```diff
@@ -99,3 +104,40 @@ fun createRunTask(
...
+    }
+
+    private var startTime = System.currentTimeMillis()
+
+    override fun beforeExecute(task: Task) {
+        startTime = System.currentTimeMillis()
+    }
+
+     override fun afterExecute(task: Task, taskState: TaskState) {
+         tasksTimes[task.name] = 1000 * (System.currentTimeMillis() - startTime)
```
Comment (olonho) [inline performance/buildSrc/src/main/kotlin/MPPTools.kt]: https://docs.oracle.com/javase/6/docs/api/java/lang/System.html#nanoTime()

File: performance/buildSrc/src/main/kotlin/MPPTools.kt [hunk 5]
```diff
@@ -60,6 +63,13 @@ fun defaultHostPreset(
         throw Exception("Host OS '$hostOs' is not supported in Kotlin/Native ${subproject.displayName}.")
 }
 
+fun getNativeProgramExtension(): String = when {
+    isMacos -> ".kexe"
+    isLinux -> ".kexe"
+    isWindows -> ".exe"
+    else -> TODO("Unknown host")
```
Comment (sbogolepov) [inline performance/buildSrc/src/main/kotlin/MPPTools.kt]: `TODO` is for WIP code that you will implement later. `error` function is more appropriate here.

File: performance/buildSrc/src/main/kotlin/MPPTools.kt [hunk 6]
```diff
@@ -60,6 +63,13 @@ fun defaultHostPreset(
...
+}
+
 // Create benchmarks json report based on information get from gradle project
 fun createJsonReport(projectProperties: Map<String, Any>): String {
     fun getValue(key: String): String = projectProperties[key] as? String ?: "uknown"
```
Comment (sbogolepov) [inline performance/buildSrc/src/main/kotlin/MPPTools.kt]: "unknown"

**Pull request merged**